dev-support/mini-submarine/Dockerfile

```diff
@@ -38,6 +38,11 @@ RUN \
   apt-get update && \
   apt-get -y install vim
 
+# Install libgomp1
```

Comment:
change to `# Install libgomp1 for MXNET`

Reply:
done